WebKitPlus.podspec

```diff
@@ -15,8 +15,8 @@ Pod::Spec.new do |s|
     s.framework = "WebKit"
     s.source = { :git => "https://github.com/yashigani/WebKitPlus.git", :tag => "#{s.version}" }
 
-    s.source_files = "WebKitPlus/*.{swift,h}"
-    s.resource_bundles = { "WebKitPlus" => ["WebKitPlus/*.lproj"] }
+    s.source_files = "Sources/WebKitPlus/**/*.{swift,h}"
+    s.resources = "Sources/WebKitPlus/Sources/Resources/**/*.lproj"
```

Comment:
Why `resource_bundles` is replaced with `resources`?

It seems that considering the case where it is built as a static library, `resource_bundles` is recommended.

- https://guides.cocoapods.org/syntax/podspec.html#resource_bundles
- https://guides.cocoapods.org/syntax/podspec.html#resources

Reply:
Right. I look good to use `resource_bundles` :+1: